pilot/docker/Dockerfile.proxy_init

```diff
@@ -1,8 +1,24 @@
-FROM ubuntu:xenial
+FROM ubuntu:xenial as builder
 RUN apt-get update && apt-get upgrade -y && apt-get install -y \
     iproute2 \
     iptables \
  && rm -rf /var/lib/apt/lists/*
 
+FROM gcr.io/distroless/base:debug as distroless
+
+FROM gcr.io/distroless/base:debug
+
+COPY --from=builder /sbin/xtables-multi /sbin/xtables-multi
+COPY --from=builder /lib/xtables /lib/xtables
+COPY --from=builder /lib/x86_64-linux-gnu /lib/x86_64-linux-gnu
+COPY --from=builder /sbin/iptables* /sbin/
+COPY --from=builder /sbin/ip6tables* /sbin/
+
+COPY --from=builder /etc/iproute2 /etc/iproute2
+COPY --from=builder /bin/ip /bin/ip
+COPY --from=builder /sbin/ip /sbin/ip
+
+COPY --from=distroless  /busybox/sh /bin/sh
```

Comment:
is copying bash from the builder image not possible here?  The busybox developer is very litigious...

Cheers
-steve